Client/mods/deathmatch/logic/CClientEffekseerEffect.cpp

```diff
@@ -0,0 +1,54 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        mods/shared_logic/CClientEffekseerEffect.cpp
+ *  PURPOSE:     Dummy entity class
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+#include <codecvt>
+
+CClientEffekseerEffect::CClientEffekseerEffect(CClientManager* pManager, ElementID ID) : ClassInit(this), CClientEntity(ID)
+{
+    SetTypeName("effekseer-effect");
+
+    m_pManager = pManager;
+    m_pEffect = nullptr;
```

Comment:
m_pEffect can be nulled in the member initializer list(or even better as a default member value).